packages/php-wasm/universal/src/lib/fs-helpers.ts

```diff
@@ -198,6 +198,93 @@ export class FSHelpers {
 		return FS.isFile(FS.lookupPath(path).node.mode);
 	}
 
+	/**
+	 * Creates a symlink in the PHP filesystem.
+	 *
+	 * @param FS
+	 * @param target
+	 * @param link
+	 */
+	static symlink(FS: Emscripten.RootFS, target: string, link: string): any {
+		return FS.symlink(target, link);
+	}
+
+	/**
+	 * Checks if a path is a symlink in the PHP filesystem.
+	 *
+	 * @param FS
+	 * @param path
+	 * @returns True if the path is a symlink, false otherwise.
+	 */
+	static isSymlink(FS: Emscripten.RootFS, path: string): boolean {
```

Comment:
Could we add an 
`@rethrowFileSystemError('Could not stat "{path}"')`
decorator here like we have on methods like `isDir()`?

There isn't one on `isFile()`, which is likely my fault. (If you don't mind, you could add on there as well...)

Reply:
Sure, I've added the decorators and also fixed some of the jsdocs.